### DISCUSSION TITLE
Intel ME Not Activated (Pre), v12.0

### DISCUSSION THREAD

**Discussion by itayiad** [Ylianst/MeshCentral#6489] [Q&A]
I am failing to connect to MeshCentral server agentless.  The client Intel Me version is 12.0.94 and the Meshcentral server is 1.132

![image](https://github.com/user-attachments/assets/d5966bc4-d2b6-4d3e-ad2b-8c182687c091)

**Comment by si458** [comment]
I'm waiting on an amt 12 device to arrive in my hands so then I can figure out why amt 12 devices seem to have so much problems!?

Have u activated the amt in the mbex and then entered the password in meshcentral?

**Reply by itayiad** [reply]
Yes I have activated in MBEX

**Reply by si458** [reply]
are you using the `Add Local` option and entering IP address/user/pass/etc manually?
OR
are you using the `Setup` option and using the `meshcmd` to activate the amt device?

**Reply by itayiad** [reply]
I am using  the Setup option

**Reply by si458** [reply]
do you get any output or errors when you run the command on the remote machine?

**Comment by itayiad** [comment]
No errors just the  message I attached above

**Reply by si458** [reply]
so no output and the device just shows up in the web ui? seems odd?
